resolwe/flow/executors/prepare.py

```diff
@@ -49,9 +49,9 @@ def extend_settings(self, data_id, files, secrets):
         """
         pass
 
-    def get_tools_paths(self):
+    def get_tools_paths(self, kubernetes=False):
```

Comment:
Name variables after the functionality, not after technology (i.e. `use_cache` or something similar).